study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/viewSpatialStudyActiveTask.jsp

```diff
@@ -26,7 +31,7 @@
         </span>
         <span
             class="ml-xs sprites_v3 filled-tooltip" data-toggle="tooltip"
-            title="This must be a human-readable activity identifier and unique across all activities of the study.Note that this field cannot be edited once the study is Launched."></span>
+            title="A human-readable identifier that must be unique across all activities of the study. Allowed characters are lowercase alphabets (a-z), digits (0-9), _ (underscore) and -(minus)."></span>
```

Comment:
change to say `... are lowercase letters (a-z), ...` and add a space between the `-` & `(minus)` at the end.

Reply:
Fixed review comment.